source/screenExplorer.py

```diff
@@ -1,8 +1,68 @@
+#screenExplorer.py
+#A part of NonVisual Desktop Access (NVDA)
+#Copyright (C) 2012-2018 NV Access Limited, Joseph Lee
+#This file is covered by the GNU General Public License.
+#See the file COPYING for more details.
+
+"""Module which implements various utilities for screen exploration.
+This includes touchscreen movement as well as handling coordinate announcement for mouse and objects.
+"""
+
 import api
 import tones
 import controlTypes
 import textInfos
 import speech
+import screenBitmap
+import config
+import mouseHandler
+import wx
+
+def playLocationCoordinates(x, y, screenWidth, screenHeight, screenMinPos=None, detectBrightness=True,blurFactor=0):
+	"""Plays a tone indicating the XY-coordinate for the given location. This works for keyboard-based object navigation/focus movement, mouse movement and during touch hover gesture.  
+	@param obj: the coordinate for the location and the screen resolution (typically desktopObject.location)
+	@type obj: int
+	"""
+	# Have a fake wx.Point in case min pos is not defined on multiple monitors.
+	if screenMinPos is None:

```

Comment:
Why should this be allowed? See my comment below about the way you calculate the screen width and height.

Reply:
Hi,

This came about due to wx.Point fix in order to support mouse tracking on multiple monitors (Reef knows more about this than I do).

Thanks.

Reply:
May be we should step away from getting this information from wx and use [GetSystemMetrics](https://msdn.microsoft.com/en-us/library/windows/desktop/ms724385(v=vs.85).aspx) instead?

Reply:
To clarify, see the SM_CXVIRTUALSCREEN, SM_CYVIRTUALSCREEN, SM_XVIRTUALSCREEN and SM_YVIRTUALSCREEN. If I understand the code correctly (and probably I don't), throwing these four constants at GetSystemMetrics would make mouseHandler.getTotalWidthAndHeightAndMinimumPosition obsolete.